Clicking on a file in tree view causes an immediate crash

Atom immediately crashes whenever I click on a file in the tree view or try to open a file when a particular git project is open in tree view.
* .ico & .gif files are fine, php, js, tpl, html, rb, and strangely .mp3 files all cause a crash

Details:
- Atom version 1.1.0
- successful repro in --safe
- Occassional traceback:
```
events.js:141
      throw er; // Unhandled 'error' event
            ^
Error: write EPIPE
    at exports._errnoException (util.js:814:11)
    at process.target._send (internal/child_process.js:552:28)
    at process.target.send (internal/child_process.js:485:12)
    at global.emit (C:\Users\B06537\AppData\Local\Atom\app-1.1.0\resources\app.a
sar\src\task-bootstrap.js:54:22)
    at Object.console.error (C:\Users\B06537\AppData\Local\Atom\app-1.1.0\resour
ces\app.asar\src\task-bootstrap.js:20:21)
    at process.<anonymous> (C:\Users\B06537\AppData\Local\Atom\app-1.1.0\resourc
es\app.asar\src\task-bootstrap.js:67:22)
    at emitOne (events.js:77:13)
    at process.emit (events.js:169:7)
    at process._fatalException (node.js:219:26)
```

This problem doesn't occur unless I've added a particular git repo folder or subfolder, but then effects every other project folder once it's been added.

Important notes:
- Context menus don't appear in tree view unless I'm right clicking a folder or file directly

![atomerr](https://cloud.githubusercontent.com/assets/5964658/10837170/ad2ee5a2-7ef9-11e5-9417-5d560ff4b25d.png)

*(After really digging into this issue, the following seems irrelevant as the real culprit may have been a surreptitious (read: automatic and completely silent) update to atom v1.1.0, but I'm leaving this in here for the sake of thoroughness)*
Activity up till the issue occured:
1  disable jshint package
2  instal linter-eslint package
3  try to use linter-eslint (throws error: linter package dependancy)
4  install linter
5  try to use linter-eslint (throws errors:
     - missing local config, using global config
     - cannot evaluate string as javascript due to unsecure-eval setting?
       * w/stack trace but no submit issue button <-- first weird problem)
6. This is about where I lost the ability to open new files in tree view, though opening them from menus works.

@joshaber I'm trying to build with your git-utils branch, but build fails with:

make: *** No rule to make target `Release/obj.target/libgit2/deps/libgit2/src/annotated_commit.o', needed by `Release/git2.a'.  Stop.

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "fs-plus": "^2.8.0",
     "fstream": "0.1.24",
     "fuzzaldrin": "^2.1",
-    "git-utils": "^4",
+    "git-utils": "^4.0.7",
     "grim": "1.5.0",
     "jasmine-json": "~0.0",
     "jasmine-tagged": "^1.1.4",

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "normalize-package-data": "^2.0.0",
     "nslog": "^3",
     "oniguruma": "^5",
-    "pathwatcher": "^6.2",
+    "pathwatcher": "~6.2",
     "property-accessors": "^1.1.3",
     "random-words": "0.0.1",
     "resolve": "^1.1.6",

